modules/kg-bbp-schemas/src/test/resources/invalid/nexus/core/mediatype/v0.1.0/wrong-type.json

```diff
@@ -0,0 +1,5 @@
+{
+  "@context":"{{base}}/contexts/nexus/core/mediatype/v0.1.0",
+  "@type":"wrong:Type",
```

Comment:
change to **nsg:wrongType**

Reply:
I'll change it to nxv:WrongType since nexus-kg is not aware of the nsg namespace.